src/index.js

```diff
@@ -0,0 +1,172 @@
+import UploaderService from './service';
+
+const KILOBYTE = 1000,
+    MEGABYTE = 1024;
+
+class UploadController {
```

Comment:
We've discussed about the name of this class but I am not sure Controller fit here, it imply that we use MVC here and we don't. and overall responsibility of Controller is to connect between model to view - which we don't have directly within this package (but within the consumer itself).

it's either `Upload` or something else that describe that here we have the main logic of our package.